src/libponyrt/sched/start.c

```diff
@@ -150,6 +165,21 @@ PONY_API int pony_init(int argc, char** argv)
 
   ponyint_cpu_init();
 
+  if (opt.threads == 0) {
+    opt.threads = ponyint_cpu_count();
+  }
+  else if (opt.threads > ponyint_cpu_count())
+  {
+    printf("Can't have --ponythreads > physical cores, the number of threads you'd be running with (%u > %u)\n", opt.threads, ponyint_cpu_count());
+    exit(-1);
+  }
+
+  if (opt.min_threads > opt.threads)
+  {
```

Comment:
small nit. this can be fixed in another PR. this `{` should be on the preceding line.

Reply:
That was the style most present in the code base. Not my preference neither… but I try to keep things consistent in a code base, if only one file. If we want `{` on the same line (my preference 😃 and which is what I understand you are asking for) should I do a single commit formatting pass throughout the file?